charts/lakom/charts/runtime/templates/deployment.yaml

```
@@ -52,21 +52,21 @@ spec:
                 - key: app.kubernetes.io/name
                   operator: In
                   values:
-                  - {{ .Values.name }}
+                  - {{ .Values.global.name }}
                 - key: app.kubernetes.io/instance
                   operator: In
                   values:
                   - {{ .Release.Name }}
               topologyKey: "kubernetes.io/hostname"
       {{- end }}
       priorityClassName: {{ .Values.priorityClass.name }}
-      {{- if or .Values.kubeconfig .Values.projectedKubeconfig }}
+      {{- if or .Values.global.kubeconfig .Values.projectedKubeconfig }}
       automountServiceAccountToken: false
       {{- else }}
-      serviceAccountName: {{ .Values.name }}
+      serviceAccountName: {{ .Values.global.name }}
       {{- end }}
       containers:
-      - name: {{ .Values.name }}
+      - name: lakom-admission-controller
```

```suggestion
      - name: {{ .Values.global.name }}
```
Is there a reason the name is not used?

---

Yes, there is a reason. The container name is usually part of the prometheus metrics and it would be useful to easily filter the metrics from all lakom containers in a cluster, this is why I changed the container name to a static name.

---

So by leaving it static you're assuring that it won't change In the future by mistake?

---

Yes

---

Thank you for the explanation